keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -439,31 +439,29 @@ Operation metrics from [csiOperationsLatencyMetric](https://github.com/kubernete
 
 While a VolumeAttributesClass is referenced by any PVC, we will prevent the object from being deleted by adding a finalizer([reference](https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/storage/storageobjectinuseprotection/admission.go)). 
 
-The VACObjectInUseProtection admission controller sets the finalizer on all VolumeAttributesClasses. VACProtectionController removes the finalizer when it's not referenced. This prevents users from deleting a VolumeAttributesClass that's used by a PVC.
+The external-provisioner and external-resizer set/remove the finalizer on all VolumeAttributesClasses. This prevents users from deleting a VolumeAttributesClass that's used by a PVC.
 
 There are a few conditions that will trigger add/remove pvc finalizers in the VolumeAttributesClass:
 
 
 1. PVC created with a VolumeAttributesClass
 
-    The **VACObjectInUseProtection admission controller**:
-    * Check if the VolumeAttributesClass exists. If not, the PVC will enter the INPROGRESS state because we do not want to impose ordering on object creation
+    The **external-provisioner**:
+    * Check if the VolumeAttributesClass exists. If not, the PVC will enter the PENDING state because we do not want to impose ordering on object creation
```

Comment:
Can you add a line somewhere to describe what happens if VolumeAttributesClassName is nil in PVC spec?

Reply:
If it is nil it will not be set in either provisioner nor resizer, let me add that.